odp/genetlink.go

```diff
@@ -18,15 +18,22 @@ func (nlmsg *NlMsgBuilder) PutGenlMsghdr(cmd uint8, version uint8) *GenlMsghdr {
 	return res
 }
 
-func (nlmsg *NlMsgParser) CheckGenlMsghdr(cmd int) (*GenlMsghdr, error) {
+func (nlmsg *NlMsgParser) CheckGenlMsghdr(cmds ...int) (*GenlMsghdr, error) {
```

Comment:
Maybe for the sake of readability it makes sense to have two explicit params instead - `CheckGenlMsghdr(cmd int, fallbackCmd *int)`?

Reply:
Sounds reasonable, will push a fix. thanks!